### PR TITLE
[peer-967] Add delay before send user tip question/answer notification

### PR DESCRIPTION
**Pull request by s-kashko**

### CODE REVIEW

File: app/containers/SendTips/saga.js [hunk 1]
```diff
@@ -173,6 +177,44 @@ export function* sendTipsWorker({ resetForm, val, questionId, answerId }) {
...
+export function* sendTipsNotificationWorker({
+  data,
+  questionId,
+  answerId,
+  transactionId,
+  block,
+}) {
+  try {
+    let attempts = 1;
+    while (attempts <= 5) {
```
Comment (sergeyi23) [inline app/containers/SendTips/saga.js:189]: Move 5 into a variable to not hardcode it twice

File: app/containers/SendTips/saga.js [hunk 2]
```diff
@@ -173,6 +177,44 @@ export function* sendTipsWorker({ resetForm, val, questionId, answerId }) {
...
+      // delay before notifications tips service call
+      yield new Promise(res => {
+        setTimeout(() => {
+          res();
+        }, 700);
```
Comment (sergeyi23) [inline app/containers/SendTips/saga.js:194]: Let's increase this timeout to 2 seconds and reduce number of attempts to 3.

**Review by sergeyi23 — APPROVED** [review]

File: app/containers/SendTips/saga.js [hunk 3]
```diff
@@ -173,6 +177,44 @@ export function* sendTipsWorker({ resetForm, val, questionId, answerId }) {
...
+
+      const result = yield call(callService, NOTIFICATIONS_TIPS_SERVICE, {
+        ..._omit(data, 'memo'),
+        questionId,
+        answerId,
+        transactionId,
+        block,
+      });
+
+      if (result.OK) break;
```
Comment (sergeyi23) [inline app/containers/SendTips/saga.js:205]: It is OK to break the cycle is result is successful. It is a valid approach.

**Review by sergeyi23 — APPROVED** [review]

File: app/containers/SendTips/saga.js (hunk 1 above)
Comment (s-kashko) [reply app/containers/SendTips/saga.js]: fixed

File: app/containers/SendTips/saga.js (hunk 2 above)
Comment (s-kashko) [reply app/containers/SendTips/saga.js]: updated

**Pull request merged**